data/free worlds start.txt

```diff
@@ -1399,6 +1399,103 @@ mission "FW Pirates 3.1B"
+				`	It also looks as if they've brought some other friends along. You notice a Republic carrier emitting strange energy signatures and launching fighters that are clearly alien in design.`
+				accept
+	on accept
+		set "fw pirates: own way"
+
+	npc kill
+		personality plunders staying
+		government "Pirate"
+		system Shaula
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Marauder fleet I"
+		fleet "Marauder fleet II"
+		fleet "Marauder fleet III"
+		fleet "Marauder fleet IV"
+		fleet "Marauder fleet V"
+		fleet "Marauder fleet VI"
+		fleet "Marauder fleet VII"
+		fleet "Marauder fleet VIII"
+		fleet "Marauder fleet IX"
+		fleet "Marauder fleet X"
+		ship "Bastion (Heavy)" "Dread"
+		ship "Carrier (Alpha)" "Trafficker"
+		ship "Marauder Falcon (Engines)" "Made Man"
+		ship "Marauder Falcon (Engines)" "Consigliare"
+		ship "Marauder Falcon (Engines)" "Capo Famiglia"
+		ship "Marauder Falcon (Engines)" "Oyabun"
+		ship "Marauder Falcon (Engines)" "Wakagashira"
+		ship "Marauder Falcon (Engines)" "Shateigashira"
+		ship "Marauder Falcon (Engines)" "Kyodai"
+		dialog "Songs will be written about your victory here today, but all you will remember is the screams of terrified, dying teenage pirate crews echoing over the radio, wave upon wave of incoming missiles, and the clouds of wreckage littering the system at the end. You shake yourself out of your reverie. You have work to do now on <destination>. It's time to finish what you've begun."
+	npc
+		government "Pirate"
+		system "Shaula"
+		personality heroic nemesis staying vindictive
+		ship "Korath Chaser" "Brutus"
+		ship "Korath Chaser" "Benedict Arnold"
+		ship "Korath Chaser" "Alcibiades"
+		ship "Korath Chaser" "Ephialtes"
+	on complete
+		set "fw pirates: crushed"
+		event "fw greenrock crushed"
+
+
+event "fw pirates crushed"
+	planet "Greenrock"
+		description `Until recently, Greenrock was the center of piracy and slavery in the south. Then a Free Worlds strike force annihilated their fleet, destroyed their support base, and freed their slaves. Without the stolen goods and slave labor to support it, the economy collapsed overnight. Small-scale wars are popping up everywhere between anarchist communities, shipless pirate gangs, and scavengers.. `
+		spaceport clear
+		shipyard clear
+		outfitter clear
+"required reputation" 100	
```

Comment:
Hah! Good point. I guess I need to clear the landing requirement, too. I'll clear its security, bribe, and tribute settings. I'll also up the clandestine pay the player gets when this branch completes in order to compensate the player for lost tribute income (was 1800 per day from Greenrock)